bcipy/helpers/demo/demo_load_raw_data.py

```diff
@@ -0,0 +1,28 @@
+from bcipy.helpers.load import (
+    load_raw_data,
+)
+from pathlib import Path
+
+
+def main(data_folder):
+    raw_data_file = 'raw_data.csv'
+    raw_data = load_raw_data(Path(data_folder, raw_data_file))
+    channels = raw_data.channels
+    type_amp = raw_data.daq_type
+    sample_rate = raw_data.sample_rate
+    # access data using raw_data.channel_data or raw_data.numerical_data
+    print('Successfully loaded raw data from {}'.format(raw_data_file))
+    print('Channels: {}'.format(channels))
+    print('Type: {}'.format(type_amp))
+    print('Sample rate: {}'.format(sample_rate))
+    print('Data shape: {}'.format(raw_data.channel_data.shape))
```

Comment:
Any reason you used `format` over `f-strings`?

Reply:
I copied it! haha. I will change to f-string I think they are more performant